### DISCUSSION TITLE
can i contribute to this project

### DISCUSSION THREAD

**Discussion by ajin-varghese-chandy** [jhasse/jngl#113] [General]
I'm interested in contributing to the project and wanted to inquire if you are open to external contributions. If so, could you please provide some guidance on how I can get started?

**Comment by jhasse** [comment]
Hi :)

Yes, I'm very open to external contributions. Feel free to have a look at anything you want to improve.

To get started I would recommend to build JNGL itself and run the `jngl-test` or `jngl-unittest`. Have you created a game with JNGL yet? [The tutorials](https://github.com/jhasse/jngl/wiki) could always use some improvements.

If you're stuck with anything, you can also come to Discord and ask me there.

**Comment by pperta** [comment]
<img width="225" alt="é" src="https://github.com/user-attachments/assets/91a51c8f-e211-4afc-9a8c-b488d585a3fd">
quest ce que ça veut dire?
